model/task/task.go

```diff
@@ -2608,8 +2609,29 @@ func GetTasksByVersion(versionID, sortBy string, statuses []string, variant stri
 		match[DisplayNameKey] = bson.M{"$regex": taskName, "$options": "i"}
 	}
 
+	const tempParentKey = "_parent"
 	pipeline := []bson.M{
 		{"$match": match},
+		// do a self join to filter off execution tasks
+		{"$lookup": bson.M{
+			"from":         Collection,
+			"localField":   IdKey,
```

Comment:
IdKey and ExecutionsTasksKey represent a string slice and and a string. will that affect the equality match?